scan-payment/src/main/java/com/getbouncer/scan/payment/ml/ssd/SSD.kt

```diff
@@ -144,3 +144,51 @@ fun filterVerticalBoxes(detectedBoxes: List<DetectionBox>): List<DetectionBox> {
 
     return detectedBoxes.filter { abs(it.rect.centerY() - medianCenter) <= medianHeight }
 }
+
+/**
+ * The model outputs a particular location or a particular class of each prior before moving
+ * on to the next prior. For instance, the model will output probabilities for background
+ * class corresponding to all priors before outputting the probability of next class for the
+ * first prior. This method serves to rearrange the output if you are using outputs from
+ * multiple layers If you use outputs from single layer use the method defined above
+ *
+ * TODO: simplify this
+ */
+fun rearrangeArray(
```

Comment:
I know this is copied directly from my code, but let's rename this to `rearrangeObjArray` to make it clear where this is used.